region/brinstar/green.json

```diff
@@ -1077,7 +1077,22 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": [
+                    {"or": [
+                      "HiJump",
+                      "h_canFly"
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Wall Jump",
+                  "notable": false,
+                  "requires": [ "canWalljump" ]
+                },
+                {
+                  "name": "Crouch Jump Down Grab",
+                  "notable": false,
+                  "requires": [ "canDownGrab" ]
                 }
```

Comment:
I guess if we want to be thorough we need a SpringBall Jump strat.

There's also a ~20-frame shinespark that can be done here if you come in charged from node 2 with ~150 frames remaining. It's annoying, you have to double-angle to aim up and shoot the shot blocks, but it's a thing you can do.

Reply:
We haven't typically considered strats with tech that is significantly harder than simple strats with the same item requirements. 
Springball jumps haven't been added in other rooms where a simple wall jump works.